packages/env/lib/commands/log.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const dockerCompose = require( 'docker-compose' );
+
+/**
+ * Internal dependencies
+ */
+const initConfig = require( '../init-config' );
+
+/**
+ * Displays the Docker & PHP logs on the given environment.
+ *
+ * @param {Object}  options
+ * @param {Object}  options.environment The environment to run the command in (develop or tests).
+ * @param {Object}  options.watch       If true, watch and continue showing logs.
+ * @param {Object}  options.spinner     A CLI spinner which indicates progress.
+ * @param {boolean} options.debug       True if debug mode is enabled.
+ */
+module.exports = async function log( { environment, watch, spinner, debug } ) {
+	const config = await initConfig( { spinner, debug } );
+
+	// If we show text while watching the logs, it will continue showing up every
+	// few lines in the logs as they happen, which isn't a good look. So only
+	// show the message if we are not watching the logs.
```

Comment:
This reminds me of something entirely unrelated that frustrates me which is that you can't pipe output from `wp-env run` into other bash commands. We should probably only ever do informational logging if we're sure that `wp-env` is running in a terminal emulator.

Reply:
😁 I don't know much about that stuff. Is there any action I can take here to improve the code?